http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2013,6 +2013,8 @@ components:
               cloudService: gce
               machineType: n1-highmem-16
               diskSize: 500
+            labelsToUpsert: {new_label : value, old_label : new_value}
```

Comment:
Is this valid yaml syntax? I usually run any swagger change through https://editor.swagger.io/

Reply:
Looks like the swagger validator is happy 👍

Reply:
yeah I've been using that validator check my changes